src/app/persistence/String.h

```diff
@@ -77,35 +78,50 @@ class ShortString
 /// Provides internal access and processing to a ShortString class.
 ///
 /// Meant for internal use only, this is NOT public API outside the SDK code itself.
-class ShortStringIO
+class ShortStringWriteIO
```

Comment:
So this is a bit confusing, in that the data is being _read_ from storage and _written_ to the ShortString, right?  And the naming seems based on the latter, but I was assuming it was based on the former and thought it was backwards.

It's especially confusing because this thing returns a ReadBuffer...

Would it be clearer to call this thing ReadIntoShortString (or just ShortStringReader) and the other class WriteShortString or ShortStringWriter?   The Reader/Writer versions still have the ambiguity: is a ShortStringReader for reading _from_ a ShortString or _into_ a ShortString?

Maybe just ShortStringReadFromStorage and ShortStringWriteToStorage?